include/wil/cppwinrt_helpers.h

```diff
@@ -226,6 +226,51 @@ struct dispatcher_traits<winrt::Microsoft::UI::Dispatching::DispatcherQueue>
 #endif // __WIL_CPPWINRT_MICROSOFT_UI_DISPATCHING_HELPERS
 /// @endcond
 
+#if defined(WINRT_Windows_Foundation_H) && !defined(__WIL_CPPWINRT_WINDOWS_FOUNDATION_BUFFER_HELPERS) ||\
+    defined(WIL_DOXYGEN)
+/// @cond
+#define __WIL_CPPWINRT_WINDOWS_FOUNDATION_BUFFER_HELPERS
+namespace Windows::Foundation
+{
+    struct IMemoryBufferByteAccess;

```

Comment:
```suggestion
struct IMemoryBufferByteAccess;
```
To make clang-format happy

Reply:
Can I on a fork?

One thing I'd really like to do if I ever got the time to do it is to add a pipeline/action/something that I can run that will run clang format & commit the result, either as a separate command (e.g. `/azp format`) or just part of the pipeline by default.